@uportal/grid-carousel/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@uportal/grid-carousel",
+  "private": false,
+  "version": "1.39.0",
+  "main": "src/grid-carousel.js",
+  "type": "module",
+  "exports": {
+    ".": {
+      "import": "./dist/grid-carousel.es.js",
+      "require": "./dist/grid-carousel.umd.cjs"
+    }
+  },
+  "engines": {
+    "node": "^14.0.0",
+    "npm": "^6.0.0"
+  },
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build && vite build --minify=false --emptyOutDir=false",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "lit": "^2.0.0",
+    "@uportal/open-id-connect": "^1.39.0",
+    "@uportal/portlet-registry-to-array": "^1.39.0",
+    "lodash.get": "^4.4.2",
+    "swiper": "^9.4.1"
+  },
+  "devDependencies": {
+    "@babel/plugin-proposal-class-properties": "^7.18.6",
+    "@babel/plugin-proposal-decorators": "^7.22.5",
+    "@babel/plugin-syntax-import-assertions": "^7.22.5",
+    "minimist": "^1.2.8",
+    "vite": "^4.3.9",
+    "vite-plugin-babel": "^1.1.3"
+  },
+  "bugs": {
+    "url": "https://github.com/uPortal-contrib/uPortal-web-components/issues"
+  },
+  "license": "Apache-2.0",
+  "author": "Apereo <uportal-dev@apereo.org>",
+  "keywords": [
+    "apereo",
+    "uportal",
+    "grid",
+    "carousel",
+    "favorites",
+    "app",
+    "web components"
```

Comment:
Does `web components` work for a keyword even though there's a space in it?

Reply:
Yes this is valid